README.md

```diff
@@ -312,6 +313,34 @@ HostedGPT requires these services to be running:
 
 Every time you pull new changes down, kill `bin/dev` and then re-run it. This will ensure your local app picks up changes to Gemfile and migrations.
 
+## Language models
+
+Each User has their own list of Language Models they can use.
+
+### Refreshing language models
+
+There is a shared list of known LLM models for OpenAI, Anthropic, and Groq in `models.yaml` and a Rake task to import them into all users:
+
+```plain
+rails models:import
+```
+
+### Update models.yaml
+
+The `models.yaml` file in the root of the project is used by HostedGPT applications to refresh their local list of models.
+
+To refresh this file using the models in local DB, run:
+
+```plain
+rails models:export
+```
+
+To export the models to another file, either json or yaml, pass in an argument:
+
+```plain
+rails models:export[tmp/models.json]
+```
```

Comment:
Thanks for adding this explanation. I am wondering though if we can add some smart defaults so that a new user can safely ignore this and still get reasonable behavior. Can we make it so that `bin/dev` also runs `models:import` on startup? It already runs migrations and bundle upon startup so I've been leaning into it.

We have a fair number of people using this app who are only mildly technical, or are technical but don't know rails, so it's been nice to keep the day-to-day running of it super simple.

But actually, now that I write all this, I don't think people will reliably restart their server. It's not like fly, render, heroku do an automatic nightly restart or anything. Now I'm on the fence... I think I still learn towards doing it on startup, since it's a bit of a catch all. But we'll also add a button to the Settings at some point. And maybe later on we could even add a nightly sync of models:import in the queue. (Scope creep for now, I'm just thinking out loud.)